### PR TITLE
Add AffGameSnakeJs

### PR DESCRIPTION
**Pull request by 3ddyy**
Currently, `purescript-game` is unreleased and not in the package set. When i've written docs, released it, and gotten it in the package set, i will remove it from the additions here so that it can be merged. Not marking it as a draft PR because i'd still say this is ready for review.

Request issue: https://github.com/JordanMartinez/purescript-cookbook/issues/232

### CODE REVIEW

**Comment by milesfrain** [comment]
Looks good.
Made a small change to just point to the original recipe (thanks for the citation though). Make sure to re-pull your branch to pickup this latest addition.

**Comment by JordanMartinez** [comment]
Yeah, I think this looks good, too.

File: packages.dhall [hunk 1]
```diff
@@ -3,6 +3,71 @@ let upstream =
 
 let overrides = {=}
 
-let additions = {=}
```
Comment (milesfrain) [inline packages.dhall:6]: Obvious to all of us, but just noting for the record that we'll merge once we can eliminate this change.

Comment (JordanMartinez) [reply packages.dhall:6]: I think we should add a tag to PRs that are basically "good to merge" once their dependencies are included in the package set. Thoughts?

Comment (milesfrain) [reply packages.dhall:6]: Sounds good. Added the `package-set` label. Feel free to edit that label if you had something else in mind.

**Comment by 3ddyy** [comment]
PR to package-sets submitted (https://github.com/purescript/package-sets/pull/678), will just have to wait for that to be merged, and then i can get this one ready.

**Comment by JordanMartinez** [comment]
Looks like the package got merged and a new release including it was made. Can you update the package set to the latest one and push the changes here? That should get the build working and then we can merge it.

**Comment by 3ddyy** [comment]
That should be it!

**Pull request merged**